src/Chord.java

```diff
@@ -0,0 +1,115 @@
+import components.queue.Queue;
+import components.queue.Queue1L;
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+
+/**
+ * Allows a user to transverse through a chord.
+ *
+ * @author
+ */
+public class Chord {
+
+    /**
+     * Removes the last note in a chord.
+     *
+     * @param q
+     *
+     * @return removed
+     */
+    public static char removeLast(Queue<Character> q) {
+        q.flip();
+        char removed = q.dequeue();
+        q.flip();
+
+        return removed;
+    }
+
+    /**
+     * Gets the root node of the given chord.
+     *
+     * @param q
+     *
+     * @return root
+     */
+    public static char getRoot(Queue<Character> q) {
+        char root = q.front();
+        return root;
+    }
+
+    /**
+     * Add the user given note to the chord.
+     *
+     * @param q
+     * @param note
+     */
+    public static void addNote(Queue<Character> q, char note) {
+        q.enqueue(note);
+    }
+
+    /**
+     * Gets the triad of the given chord.
+     *
+     * @param q
+     *
+     * @return triad
+     */
+    public static String getTriad(Queue<Character> q) {
+        String triad = "";
+        for (char y : q) {
+            String currChar = y + "";
+            if (!(triad.contains(currChar))) {
+                triad += y;
+            }
+        }
+        return triad;
+    }
+
+    /**
+     * Main Method.
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+        SimpleWriter out = new SimpleWriter1L();
+        Queue<Character> aMaj = new Queue1L<>();
```

Comment:
Looks like you proved your component statically. I think that's fine, but I would store the queue at the top of the file as a field.